tutorials/independence.py

```diff
@@ -141,6 +141,33 @@
 #
 # ------------
 #
+# **D-variable Hilbert Schmidt Independence Criterion (Dhsic)** is an extension of Hsic,
```

Comment:
dHsic is in a different module, so should be in a different tutorial specific for d-variate tests. Should be similar to the k-sample tutorial